### PR TITLE
Document how to backport a rust-lang/rust PR

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: ferrocene/doc/internal-procedures/src/backports.rst [hunk 1]
```diff
@@ -104,3 +104,29 @@ GitHub for review.
...
+   # Alternatively, add it as an ssh remote
+   # git remote add upstream-rust git@github.com:rust-lang/rust.git
+   git fetch upstream-rust
+
+Create a branch targeting the ``release/1.*`` branch you want to
+backport to, and run:
+
+.. code-block::
+
+   GITHUB_REPOSITORY="rust-lang/rust" ferrocene/tools/backport/one.py PR_NUMBER
```
Comment (pietroalbini) [inline ferrocene/doc/internal-procedures/src/backports.rst:126]: Would be nice to change the script to do:

```
ferrocene/tools/backport/one.py --rust PR_NUMBER
```

File: ferrocene/doc/internal-procedures/src/backports.rst [hunk 2]
```diff
@@ -104,3 +104,29 @@ GitHub for review.
...
+
+To backport a PR from ``rust-lang/rust``, a bit of initial one-time preparation is needed. You may
+skip the following step if it you have already done this once in your ferrocene checkout.
+
+.. code-block::
+   # Add a remote to rust-lang/rust
+   git remote add upstream-rust https://github.com/rust-lang/rust.git
+   # Alternatively, add it as an ssh remote
+   # git remote add upstream-rust git@github.com:rust-lang/rust.git
+   git fetch upstream-rust
```
Comment (pietroalbini) [inline ferrocene/doc/internal-procedures/src/backports.rst:119]: We can avoid this by having the backport script fetch the commit it's going to backport before running.

**Review by pietroalbini — CHANGES_REQUESTED** [review]
Would be nice to change the backport script to simplify the workflow here.

File: ferrocene/doc/internal-procedures/src/backports.rst (hunk 2 above)
Comment (Veykril) [reply ferrocene/doc/internal-procedures/src/backports.rst:119]: Oh, that is a good idea

**Review by pietroalbini — APPROVED** [review]
bors merge

**Comment by Veykril** [comment]
Added a newline to the codeblock, that should fix the CI failure

**Comment by Urhengulas** [comment]
bors merge

**Review by Urhengulas — APPROVED** [review]
bors merge

**Pull request merged**